<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Admin_add_btn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_Author_textBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_SourceTypeTxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_QuoteTxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_SearchBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_GenreTxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_TitleTxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_SourceTxtbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_Load">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Load All</string>
   </property>
  </widget>
  <widget class="QTableView" name="Admin_QuotesTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>701</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_Search_TextBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>701</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_DeleteBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>iQuotes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_AdminBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_AboutBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_HomeBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>901</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_EditBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_ConfirmDltBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_CancelBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>400</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Author :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quote :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>460</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Genre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>490</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Title :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Source :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>550</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Source Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Admin_AElbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>191</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_ConfirmAddBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_FindBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Admin_AuthorID_textBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Admin_ConfirmEditBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>Admin_add_btn</zorder>
  <zorder>Admin_Author_textBox</zorder>
  <zorder>Admin_SourceTypeTxtbox</zorder>
  <zorder>Admin_QuoteTxtbox</zorder>
  <zorder>Admin_SearchBtn</zorder>
  <zorder>Admin_GenreTxtbox</zorder>
  <zorder>Admin_TitleTxtbox</zorder>
  <zorder>Admin_SourceTxtbox</zorder>
  <zorder>Admin_Load</zorder>
  <zorder>Admin_QuotesTable</zorder>
  <zorder>Admin_Search_TextBox</zorder>
  <zorder>Admin_DeleteBtn</zorder>
  <zorder>label</zorder>
  <zorder>Admin_AdminBtn</zorder>
  <zorder>Admin_AboutBtn</zorder>
  <zorder>Admin_HomeBtn</zorder>
  <zorder>Admin_EditBtn</zorder>
  <zorder>Admin_ConfirmDltBtn</zorder>
  <zorder>Admin_CancelBtn</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>Admin_AElbl</zorder>
  <zorder>Admin_ConfirmAddBtn</zorder>
  <zorder>Admin_FindBtn</zorder>
  <zorder>Admin_AuthorID_textBox</zorder>
  <zorder>Admin_ConfirmEditBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
